src/Splunk.ModularInputs/Splunk/ModularInputs/ModularInput.cs

```diff
@@ -297,7 +297,9 @@ private static bool IsValidateArguments(string[] args, DebuggerAttachPoints atta
                     try
                     {
                         inputDoc = await stdin.ReadToEndAsync();
-                        writer.LogAsync(Severity.Debug, inputDoc).Wait();
+                        inputDoc = inputDoc.Replace(Environment.NewLine, " |");
```

Comment:
Yeah, so this could become something like `inputDoc = Utils.ReplaceNewLinesWithPipes(inputDoc);`

Up to you :)